.gitignore

```diff
@@ -25,6 +25,7 @@ Thumbs.db
 npm-debug.log
 npm-debug.log.*
 package-lock.json
+yarn.lock
```

Comment:
Should this be ignored?  The build is using yarn atm.  Should we switch to npm to be consistent with Cesium? https://github.com/PropellerAero/cesium/blob/master/deployment/publish.sh#L18

Reply:
That actually came from cesium source i believe